src/bootstrap.php

```diff
@@ -779,18 +779,26 @@ function (Accept $type, string $number) use ($app) {
         if (false === isset($app['articles'][$number])) {
             throw new NotFoundHttpException('Article not found');
         }
-
         $article = $app['articles'][$number];
-
+        $headers = ['Content-Type' => $type->getNormalizedValue()];
         $content = $article['relatedArticles'] ?? [];
 
+        if ($type->getParameter('version') < 2) {
+            $headers['Warning'] = sprintf('299 elifesciences.org "Deprecation: Support for version %d will be removed"', $type->getParameter('version'));
+
+            if ('13410' === $number) {
+                throw new NotAcceptableHttpException('This blog article requires version 2.');
```

Comment:
The message needs to be changed to something appropriate to this response.

Perhaps: 'These article relations require version 2.'